lib/ansible/modules/cloud/vmware/vmware_drs_rule.py

```diff
@@ -0,0 +1,288 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2017, Abhijeet Kasurde <akasurde@redhat.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+
+DOCUMENTATION = '''
+---
+module: vmware_drs_rule
```

Comment:
As this is VM-to-VM DRS rules, do we want to name this module something like `vmware_vm_vm_drs_rule`, so we can later create a `vmware_host_vm_drs_rule` module?

Reply:
I am not familiar with VM-HOST DRS rule, but later I or someone else can add that facility in this module. I like the idea of `vmware_vm_vm_drs_rule` naming though.